dali/pipeline/operators/resize/resize.cc

```diff
@@ -33,7 +33,18 @@ DALI_SCHEMA(ResizeAttr)
       "The op will keep the aspect ratio of the original image.", 0.f, true)
   .AddOptionalArg("resize_longer", "The length of the longer dimension of the resized image. "
       "This option is mutually exclusive with `resize_shorter`,`resize_x` and `resize_y`. "
-      "The op will keep the aspect ratio of the original image.", 0.f, true);
+      "The op will keep the aspect ratio of the original image.", 0.f, true)
+  .AddOptionalArg("max_size", R"code("Maximum size of the longer dimension when resizing with `resize_shorter`.
```

Comment:
Please also add note to `resize_shorter` as well about `max_size` argument.

Reply:
Done